doc/new-private-repository-azure.md

```diff
@@ -0,0 +1,517 @@
+| Yes      | AzResourceGroup    | The Resource Group that will be used to contain the Azure resources.                                                       |
+| No       | AzSubscriptionName | The name of the Azure Subscription that will be used to pay for the Azure resources.                                       |
+| No       | AzLocation         | The Azure location where the Azure resources will be created. (Default: westus)                                            |
+| No       | WorkingDirectory   | The folder location that contains this the ARM template files, as well as where the Azure Parameter files will be created. |
+
+The PowerShell script has been configured to work with `Get-Help` providing further details about the script, as well as examples of how to use it.
+
+**How to:**
+
+1. Download a local copy of the Windows Package Manager Rest Source from GitHub (github: [winget-cli-restsource](https://github.com/microsoft/winget-cli-restsource))
+    1. Select *Code* from within GitHub.
+    1. Select *Download ZIP* from the drop-down menu.
+    1. Extract the newly downloaded ZIP file to *C:\Projects\\*
+
+1. Open PowerShell, and run the following command:
+    ```powershell
+    PS C:\> C:\Projects\winget-cli-restsource\src\WinGet.RestSource.Infrastructure\automation.ps1 -ResourcePrefix "contoso" -Index "demo" -AzResourceGroup "WinGet_PrivateRepository"
+    ```
+
+1. When the script completes, it'll display a command that can be used to include the private source to your Windows Package Manager client.
+
+
+## Automatically add manifests to the private source
+
+After a Windows Package Manager private source has been setup, the next step is to add application manifests to it. 
+
+**How to:**
+
+1. Open an administrative PowerShell Window
+1. Run the following command: `Set-ExecutionPolicy Unrestricted`
+1. Load the Function library into memory by running the following `. C:\Projects\winget-cli-restsource\Tools\PrivateRepoLib.ps1`
+1. Run the following command to add the new manifest: `New-WinGetManifest -PrivateRepoName "PrivateRepo" -ManifestFilePath "C:\Temp\App.json"`
+
+> ![note]
+> For more information on how to use the `New-WinGetManifest` PowerShell function run `Get-Help New-WinGetManifest -Full` after loading the `PrivateRepoLib.ps1` script library into memory.
+
+
+## Manually create a private source
+
+The following instructions assumes the following Azure objects are named as follows:
+| Azure Resource          | Value                      |
+|-------------------------|----------------------------|
+| Azure Resource Group    | WinGet_PrivateRepo         |
+| Azure Subscription Name | Contoso Azure Subscription |
+| Azure Location          | West US                    |
+| Azure Key Vault Name    | contoso-keyvault-demo      |
+
+*We assume that the Resource Group and Subscription are pre-existing in your Azure Tenant.*
```

Comment:
This statement is confusing, I would add, something about that the key vault will be created below.  Also, why are you listing the key vault and not any of the other things which are needed like cosmos/storage/appinsights?